tags/views.py

```diff
@@ -0,0 +1,7 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+# Create your views here.
+
+def index():
```

Comment:
could we implement a simple listing API? you can refer to the method i implemented in packages